app/server/lib/sendAppPage.ts

```diff
@@ -134,6 +134,7 @@ export function makeGristConfig(options: MakeGristConfigOptions): GristLoadConfi
     notifierEnabled: server?.hasNotifier(),
     featureNotifications: isAffirmative(process.env.GRIST_TEST_ENABLE_NOTIFICATIONS),
     formFraming: GRIST_FEATURE_FORM_FRAMING as FormFraming,
+    adminDefinedUrls: process.env.GRIST_CUSTOM_COMMON_URLS,
```

Comment:
Hmm so the urls are passed as an uninterpreted string. Does that mean if there is an error, the first time we see it is on individual clients, as they attempt to interpret the string? If so, an alternative could be to parse it on the back end, so servers would fail and a deployment would abort because servers don't become healthy.